grafana-plugin/src/containers/OutgoingWebhook2Form/OutgoingWebhook2Form.tsx

```diff
@@ -49,35 +72,167 @@ const OutgoingWebhook2Form = observer((props: OutgoingWebhook2FormProps) => {
     [id]
   );
 
+  if (
+    (action === WebhookFormActionType.EDIT_SETTINGS || action === WebhookFormActionType.VIEW_LAST_RUN) &&
+    !outgoingWebhook2Store.items[id]
+  ) {
+    // nothing to show if we open invalid ID for edit/last_run
+    return null;
+  }
+
+  if (action === WebhookFormActionType.NEW || action === WebhookFormActionType.COPY) {
+    // show just the creation form, not the tabs
```

Comment:
mabe just make LastRun tab disabled for those cases?

Reply:
This follows the figma where on new/copy there's no tabs shown, plus the content is slightly different.